blog-csoar/2024-03-26-application-update.md

```diff
@@ -0,0 +1,30 @@
+---
+title: March 26, 2024 - Application Update
+keywords:
+  - sumo logic
```

Comment:
```suggestion
```

Reply:
I see, but that keyword is already present in other content 🤔